test-containers.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
@wei-lee @mikenairn Will it always make sense to add product tests with a PR to the integreatly-operator? It might make more sense to have a separate repo for the product teams to add a PR against to decouple the integreatly operator and its core tests from the product tests.

Reply:
I can't say always, but at least for now I don't think there's benefit to store this in a separate repo. It will make it more difficult to run both locally or in the pipeline, without many obvious upside.  I don't see a problem with this coupled to the integreatly-operator. It should be, as different versions of integreatly operator may require different versions of test containers. Also if we keep it in the same repo, in the future, when this file is updated, we can run e2e tests against the new test images without any additional work.